cmds/generate_timestamp.go

```diff
@@ -0,0 +1,52 @@
+package cmds
+
+import (
+	"io/ioutil"
+	"os"
+	"os/user"
+	"path/filepath"
+	"time"
+
+	"github.com/spf13/cobra"
+)
+
+var generateTimestampDoc = `
+Write the current timestamp to the file '~/.testtrack/build_timestamp.txt'.
+This timestamp can be passed as a param by the Testtrack client when calling the
+split registry endpoint from the Testtrack server.
+`
+
+func init() {
+	rootCmd.AddCommand(generateTimestampCmd)
+}
+
+var generateTimestampCmd = &cobra.Command{
+	Use:   "generate_timestamp",
+	Short: "Write the current timstamp to '~/.testtrack/build_timestamp.txt'",
+	Long:  generateTimestampDoc,
+	Args:  cobra.NoArgs,
+	RunE: func(cmd *cobra.Command, args []string) error {
+		return generateTimestamp()
+	},
+}
+
+func generateTimestamp() error {
+	usr, _ := user.Current()
+	TimestampDir := filepath.Join(usr.HomeDir, ".testtrack")
```

Comment:
I don't think we want this in the user's home directory. we want it in the project directory along side the migrations. that way this file will be get picked up when we package up the app.